### PR TITLE
Fix eslint alert for iOS sample file

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
[Bug 16846085: [eslint:Error]: @microsoft/sdl/no-postmessage-star-origin (in immersive-reader-sdk/js/samples/ios/quickstart-swift/Resources/iFr... - Boards (visualstudio.com)](https://msazure.visualstudio.com/One/_workitems/edit/16846085/)

[Bug 16845397: [eslint:Error]: @microsoft/sdl/no-postmessage-star-origin (in immersive-reader-sdk/js/samples/ios/quickstart-swift/Resources/iFr... - Boards (visualstudio.com)](https://msazure.visualstudio.com/DefaultCollection/Cognitive%20Services/_workitems/edit/16845397/)

### CODE REVIEW

File: js/samples/ios/quickstart-swift/Resources/iFrameMessaging.js [hunk 1]
```diff
@@ -17,5 +17,5 @@ window.addEventListener("message", function(message) {
 });
 
 function sendContentToReader(message) {
-    document.getElementById('immersiveReaderIframe').contentWindow.postMessage(JSON.stringify({messageType:'Content', messageValue: message}), '*');
+    document.getElementById('immersiveReaderIframe').contentWindow.postMessage(JSON.stringify({messageType:'Content', messageValue: message}), '*'); //Always specify an exact target origin, not *, when you use postMessage to send data to other windows.
```
Comment (sharad-sharma-msft) [inline js/samples/ios/quickstart-swift/Resources/iFrameMessaging.js:20]: nit: put comment on top, on the side it looks more like commented code.

Comment (sharad-sharma-msft) [inline js/samples/ios/quickstart-swift/Resources/iFrameMessaging.js:20]: always-> It is more secure to

**Review by sharad-sharma-msft — CHANGES_REQUESTED** [review]

File: js/samples/ios/quickstart-swift/Resources/iFrameMessaging.js (hunk 1 above)
Comment (cristobal-buenrostro) [reply js/samples/ios/quickstart-swift/Resources/iFrameMessaging.js:20]: done

Comment (cristobal-buenrostro) [reply js/samples/ios/quickstart-swift/Resources/iFrameMessaging.js:20]: done

**Review by sharad-sharma-msft — APPROVED** [review]

**Pull request merged**